### DISCUSSION TITLE
Troubleshooting SSL MISMATCH Error: Challenges in Connecting a Domain to GitHub and Configuring SSL on a New VPS

### DISCUSSION THREAD

**Discussion by devfemibadmus** [community/community#86693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A few months ago, I successfully connected my domain.com to my GitHub page, and everything was functioning well. However, I recently acquired a new VPS to host my website. In the process, I configured the VPS and disconnected my domain from GitHub, including its DNS settings. Subsequently, I installed a wildcard SSL for my domain, but encountered an SSL MISMATCH error.

Interestingly, the same SSL certificate worked seamlessly for my subdomains, but the main domain continued to experience issues. In an attempt to resolve the problem, I updated my DNS settings and reconnected the domain to my GitHub page. This resolved the issue with SSL, and the website now works fine without any SSL-related problems.

During the reconnection process on GitHub, I received a notification indicating the presence of an SSL certificate, which was automatically detected and completed. Despite this, I am still facing challenges with the SSL MISMATCH error for the main domain. I would greatly appreciate your assistance in resolving this matter.

**Comment by ibra-kdbra** [comment]
- Check Your SSL Certificate: The first step is to perform an SSL check on the certificate that is installed on your [site](https://kinsta.com/knowledgebase/err_ssl_version_or_cipher_mismatch/). There are a few free SSL check tools available online that you can use.
- Clear the SSL State In Your Browser: Sometimes, clearing the SSL state in your browser can resolve this issue.

**Comment by devfemibadmus** [comment]
Done I found out it’s the cache after 6hrs everything is working fine

**Reply by ibra-kdbra** [reply]
### Nice, Good Luck.
If you found the answer as helpful, please mark this question as solved.
